src/Uno.UWP/Generated/3.0.0.0/Windows.System/MemoryManager.cs

```diff
@@ -27,15 +40,26 @@ public static ulong AppMemoryUsage
 			}
 		}
 		#endif
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
-		[global::Uno.NotImplemented("__ANDROID__", "__IOS__", "NET461", "__WASM__", "__SKIA__", "__NETSTD_REFERENCE__", "__MACOS__")]
+		#if __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
+		[global::Uno.NotImplemented("__IOS__", "NET461", "__WASM__", "__SKIA__", "__NETSTD_REFERENCE__", "__MACOS__")]
 		public static ulong AppMemoryUsageLimit
 		{
 			get
 			{
 				throw new global::System.NotImplementedException("The member ulong MemoryManager.AppMemoryUsageLimit is not implemented in Uno.");
 			}
 		}
+		#else
+		public static ulong AppMemoryUsageLimit
+		{
+			get
+			{
+				ActivityManager.MemoryInfo memoryInfo = new ActivityManager.MemoryInfo();
+				ActivityManager.FromContext(ContextHelper.Current)?.GetMemoryInfo(memoryInfo);
+
+				return AppMemoryUsage + (ulong) memoryInfo.AvailMem - (ulong) memoryInfo.Threshold;
+			}
+		}
```

Comment:
We generally remove the code from the "generated" folder, as this code is generated by a tool. You'll add the code in another file named `MemoryManager.android.cs`, out of the generated folder, similar to this one. Only add this property's implementation to the file.